db/.sqlx/query-1fe4eb033e38d873d0a875093c4bfaea0f8d02c8a70744bc76cb067fee6e3a5c.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I think we probably can/want to remove this files in `db/.sqlx` and 'prefer' the one at the root level in `.sqlx`!

I think the different is these ones were likely generated without the `--workspace` option!

If we use the `--workspace` option it should look in all the crates for put the file in the workspace root

Since we are doing all the sql calls in this DB crate it matters less, but we started with the workspace version so likely good to continue with that!